x-pack/test/functional/services/cases/list.ts

```diff
@@ -81,9 +81,6 @@ export function CasesTableServiceProvider(
         rows = await find.allByCssSelector('[data-test-subj*="cases-table-row-"', 100);
         if (rows.length > 0) {
           await this.bulkDeleteAllCases();
-          // wait for a second
-          await new Promise((r) => setTimeout(r, 1000));
```

Comment:
@cnasikas this felt like it was slowly down the tests. Do we need to 1 second wait if we're going to call `waitUntilLoadingHasFinished` anyway?

Also the call to `header.waitUntilLoadingHasFinished();` was happening twice I think, once at like 86 and then it loops since `rows.length > 0` to get in this if block and again at line 79 above. So I just removed 86 since it'll happen above anyway.

Reply:
I am not sure why the timeout was there. I agree with your changes. The flaky test runner will verify that it is ok to remove it.